pandas/tests/series/test_combine_concat.py

```diff
@@ -181,7 +181,7 @@ def test_concat_empty_series_dtypes(self):
         # categorical
         assert pd.concat([Series(dtype='category'),
                           Series(dtype='category')]).dtype == 'category'
-        assert pd.concat([Series(dtype='category'),
+        assert pd.concat([Series(np.array([]), dtype='category'),
```

Comment:
See #18515

Reply:
maybe add this issue number here as well